radiation/python/config.py

```diff
@@ -35,10 +36,13 @@
 shape_nlp1 = (npts, 1, nlp1)
 shape_nlp2 = (npts, 1, nlp1 + 1)
 default_origin = (0, 0, 0)
-type_nbands = (np.float64, (nbands,))
-type_ngptlw = (np.float64, (ngptlw,))
-type_nbands3 = (np.float64, (nbands, 3))
-type_maxgas = (np.float64, (maxgas,))
-type_maxxsec = (np.float64, (maxxsec,))
-type_9 = (np.float64, (9,))
-type_10 = (np.float64, (10,))
+type_nbands = (DTYPE_FLT, (nbands,))
+type_nbandssw_flt = (DTYPE_FLT, (nbhgh - nblow + 1,))
+type_nbandssw_int = (DTYPE_INT, (nbhgh - nblow + 1,))
```

Comment:
I don't really understand what nbhgh and nblow are -- I see you have them documented in radsw_param, yay!